apis/go.mod

```diff
@@ -0,0 +1,40 @@
+module github.com/gardener/landscaper-service/apis
```

Comment:
i think in this case we could only have one single go.mod in the root of the repository.
In the landscaper case we only have 2 components because the landscaper main pkg has a lot fo dependencies which are also imported when the apis are used in other go components.

But for simplicity I would start with a single go mod.

Reply:
Ok, makes sense.